pkg/apis/deviceplugin/v1/fpgadeviceplugin_types.go

```diff
@@ -0,0 +1,95 @@
+// Copyright 2020 Intel Corporation. All Rights Reserved.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package v1
+
+import (
+	v1 "k8s.io/api/core/v1"
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+)
+
+// NOTE: json tags are required.  Any new fields you add must have json tags for the fields to be serialized.
+
+// FpgaDevicePluginSpec defines the desired state of FpgaDevicePlugin.
+type FpgaDevicePluginSpec struct {
+	// Important: Run "make generate" to regenerate code after modifying this file
+
+	// Image is a container image with FPGA device plugin executable.
```

Comment:
Should we split the image version to its own entry?

Reply:
Personally I'd like to see image names and versions in one line whereever possible. But I don't have strong opinion.

Reply:
me neither at this point. I noticed in the original doc I drafted `Version` was split out but I'm OK with this